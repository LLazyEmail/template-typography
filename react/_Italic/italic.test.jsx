import React from 'react';
import { create } from 'react-test-renderer';
import ReactDOM from 'react-dom';
import Italic from './index';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });


describe('italic/Italic component', () => {
    
    it('<Italic > has an <i> element', () => {
        const wrapper = shallow(<Italic />);
        expect(wrapper.html()).toContain('i');
    });

    test('renders without crashing', () => {
      const div = document.createElement('div');
      ReactDOM.render(<Italic />, div);
      ReactDOM.unmountComponentAtNode(div);
    });
  
    test('renders correctly', () => {
      const tree = create(<Italic><h1>Some text</h1></Italic>);
      expect(tree.toJSON()).toMatchSnapshot();
    });
});