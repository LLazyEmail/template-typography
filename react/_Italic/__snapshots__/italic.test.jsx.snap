// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`italic/Italic component renders correctly 1`] = `
<i>
  <h1>
    Some text
  </h1>
</i>
`;
